src/components/button/button.component.js

```diff
@@ -164,6 +197,10 @@ Button.propTypes = {
   fullWidth: PropTypes.bool,
   /** If provided, the text inside a button will not wrap */
   noWrap: PropTypes.bool,
+  /** Provides a tooltip message when the icon is hovered. */
+  iconTooltipMessage: PropTypes.string,
+  /** Provides positioning when the tooltip is displayed. */
+  iconTooltipPosition: PropTypes.string,
```

Comment:
As we've talked about, this type should be oneOf left right bottom top etc

Reply:
✅